voluncheer/opportunityboard/models.py

```diff
@@ -67,6 +68,7 @@ class Opportunity(models.Model):
         longitude: used for mapping the opportunity. *allowed to be blank for now
         latitude: used for mapping the opportunity. *allowed to be blank for now
         staffing: the requested number of volunteers for the opportunity.
+        volunteer: the list of registered users to the event.
```

Comment:
If it is more than one object we should be using the plural for the attribute. e.g. `volunteers`

Reply:
Changed